src/Rxjs.res

```diff
@@ -92,12 +92,12 @@ external toObservable: t<'c, 's, 'a> => t<foreign, void, 'a> = "%identity"
 @module("rxjs") external buffer: (t<'ca, 'sa, 'a>, . t<'cb, 'sb, 'b>) => t<'cb, 'sb, array<'b>> = "buffer"
 @module("rxjs") external catchError: (err => t<'class, 'source, 'a>, . t<'class, 'source, 'a>) => t<'class, 'source, 'a> = "catchError"
 
-@module("rxjs") external combineLatest2: (t<'ca, 'sa, 'a>, t<'cb, 'sb, 'b>) => t<foreign, void, ('a, 'b)> = "combineLatest"
-@module("rxjs") external combineLatest3: (t<'ca, 'sa, 'a>, t<'cb, 'sb, 'b>, t<'cc, 'sc, 'c>) => t<foreign, void, ('a, 'b, 'c)> = "combineLatest"
-@module("rxjs") external combineLatest4: (t<'ca, 'sa, 'a>, t<'cb, 'sb, 'b>, t<'cc, 'sc, 'c>, t<'cd, 'sd, 'd>) => t<foreign, void, ('a, 'b, 'c, 'd)> = "combineLatest"
-@module("rxjs") external combineLatest5: (t<'ca, 'sa, 'a>, t<'cb, 'sb, 'b>, t<'cc, 'sc, 'c>, t<'cd, 'sd, 'd>, t<'ce, 'se, 'e>) => t<foreign, void, ('a, 'b, 'c, 'd, 'e)> = "combineLatest"
-@module("rxjs") external combineLatest6: (t<'ca, 'sa, 'a>, t<'cb, 'sb, 'b>, t<'cc, 'sc, 'c>, t<'cd, 'sd, 'd>, t<'ce, 'se, 'e>, t<'cf, 'sf, 'f>) => t<foreign, void, ('a, 'b, 'c, 'd, 'e, 'f)> = "combineLatest"
-@module("rxjs") external combineLatest7: (t<'ca, 'sa, 'a>, t<'cb, 'sb, 'b>, t<'cc, 'sc, 'c>, t<'cd, 'sd, 'd>, t<'ce, 'se, 'e>, t<'cf, 'sf, 'f>, t<'cg, 'sg, 'g>) => t<foreign, void, ('a, 'b, 'c, 'd, 'e, 'f, 'g)> = "combineLatest"
+@module("rxjs") external combineLatest2: ((t<'ca, 'sa, 'a>, t<'cb, 'sb, 'b>)) => t<foreign, void, ('a, 'b)> = "combineLatest"
```

Comment:
Can you explain your change here?
The docs and examples show this as a variadic function, not one taking an array?

Reply:
Sorry for that, I was carried by a deprecation warning in the browser console and the sources passed as an array do not prints the warning, but it turned out that the sources in an array are just [one overload of `combineLatest`](https://github.com/ReactiveX/rxjs/blob/9db65635b0b26b25c35b3470885c6f02abd54122/src/internal/operators/combineLatest.ts#L15) and the deprecation relates to the `combineLatest` function in general.

Since this presents a breaking change, it makes no sense to merge that in.